### DISCUSSION TITLE
%E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/Systems/%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F/UCBCS162/

### DISCUSSION THREAD

**Discussion by giscus** [MingLLuo/CS-Plan#13] [Announcements]
# %E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/Systems/%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F/UCBCS162/

https://cs-plan.com/%E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/Systems/%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F/UCBCS162/

**Comment by lingyu2001** [comment]
求2024Spring 的视频
